src/phaser-game/utils/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const PLAYER_SPRITE_WIDTH = 32;
+export const PLAYER_SPRITE_HEIGHT = 32;
+export const PLAYER_HEIGHT = 32;
+export const PLAYER_WIDTH = 32;
+export const PLAYER_SPEED = 2;
```

Comment:
Damn, all these constants 32, but player_speed only 2?
You don't want no fastboi? xd